services-camel/src/main/java/edu/unc/lib/dl/services/camel/fulltext/FulltextProcessor.java

```diff
@@ -77,7 +63,12 @@ public void process(Exchange exchange) throws Exception {
 
         String binaryUri = (String) in.getHeader(FCREPO_URI);
         String binaryPath = (String) in.getHeader(CdrBinaryPath);
+        String fcrepoBinaryUri = (String) in.getHeader("CamelFcrepoUri");
```

Comment:
`FCREPO_URI` is actually a constant for "CamelFcrepoUri", so you could remove this and use `binaryUri` from line 64 instead